src/lib/utils/medallia.ts

```diff
@@ -7,46 +7,101 @@ type KampyleOnsiteSdk = Window &
     vaSurvey: string
   }
 
-const vagovstagingsurveys = {
-  '/search': 20,
-  '/contact-us/virtual-agent': 26,
+const SURVEY_NUMBERS = {
```

Comment:
Reverted this logic change to what is currently in production. Reminder that we don't have a way to test this until it gets to staging (known limitation with Medallia in environments).